contracts/lpp/src/lpp.rs

```diff
@@ -679,14 +740,15 @@ mod test {
 
         grant_admin_access(deps.as_mut(), &admin);
         Config::new(
-            balance_mock.denom,
+            String::from(TheCurrency::BANK_SYMBOL),
```

Comment:
[everywhere] please use `Into::into()` where possible. Specifying types as in this case clutter the code.